app/javascript/packs/brave_rewards_banner_display.jsx

```diff
@@ -50,10 +50,15 @@ class BraveRewardsPageForm extends React.Component {
   }
 
   handleDonationAmountsChange(event) {
-    this.setState({
-      donationAmounts:
-        document.getElementById("donation-amounts-input").value.split(',').map(Number)
-    });
+     let donationAmounts = document.getElementById("donation-amounts-input").value.split(',').map(Number)
+
+     donationAmounts.forEach(function(amount){
+       if(isNaN(amount)){
+         donationAmounts = [1, 5, 10]
```

Comment:
From a UX perspective, this is weird. I think the easier solution is to support strings or not verifying until the user is done typing or verifying upon submit.